pkg/apis/pipeline/images.go

```diff
@@ -36,7 +36,7 @@ type Images struct {
 	KubeconfigWriterImage string
 	// ShellImage is the container image containing bash shell.
 	ShellImage string
-	// GsutilImage is the container miage containing gsutil.
+	// GsutilImage is the container  containing gsutil.
```

Comment:
great catch @linzhaoming, can we fix the typo and have the comment with "image"?

Reply:
done

Reply:
thanks